sync-repositories.sh

```diff
@@ -1,51 +1,50 @@
 #!/usr/bin/env bash
 
-MAIN_REPO_URL=git@github.com:sequelize/sequelize.git
+MAIN_REPO_URL=https://github.com/sequelize/sequelize.git
+ROOT_DIR="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+DOCS_DIR="$ROOT_DIR/.sequelize"
+API_DIR="$ROOT_DIR/static/api"
 
 set -e
 
+# Ensure API_DIR
+mkdir -p "$API_DIR"
+
 # ======================================
 #  Sequelize v7 (typedoc + docusaurus)
 #  Cloned to its own folder because the documentation can import some of its files at build time
 # ======================================
 
-# download v7 branch
-rm -rf .sequelize/v7
-rm -rf static/api/v7
-
-git clone -b feature/typedoc --single-branch "$MAIN_REPO_URL" --depth 1 .sequelize/v7
-cd .sequelize/v7
-
-# install branch deps & build
-yarn
-
-# generate v7 typedocs
-yarn docs
+# Clear generated directories
+rm -rf "$DOCS_DIR/v7"
+rm -rf "$API_DIR/v7"
 
-mv .typedoc-build ../../static/api/v7
+# Get sources
+git clone -b feature/typedoc --single-branch "$MAIN_REPO_URL" --depth 1 "$DOCS_DIR/v7"
 
-cd ../..
+pushd "$DOCS_DIR/v7"
+yarn # Install branch deps & build
+yarn docs # Generate v7 typedocs
+mv .typedoc-build "$API_DIR/v7" # Move compiled api reference
+popd
 
 # ======================================
 #  Sequelize v6 (esdoc + docusaurus)
 #  Cloned to its own folder because the documentation can import some of its files at build time
 # ======================================
 
-# download v6 branch
-rm -rf .sequelize/v6
-rm -rf static/api/v6
-git clone -b v6 --single-branch "$MAIN_REPO_URL" --depth 1 .sequelize/v6
-cd .sequelize/v6
-
-# install branch deps & build
-yarn
+# Clear generated directories
+rm -rf "$DOCS_DIR/v6"
+rm -rf "$API_DIR/v6"
 
-# generate v6 legacy esdoc
-yarn docs
-
-mv esdoc ../../static/api/v6
+# Get sources
+git clone -b v6 --single-branch "$MAIN_REPO_URL" --depth 1 .sequelize/v6
 
-cd ../..
+pushd "$DOCS_DIR/v6"
+yarn # Install branch deps & build
+yarn docs # Generate v7 typedocs
```

Comment:
For when the file will be changed next, we should fix this copy paste error

Reply:
🤦

Reply:
https://github.com/sequelize/website/blob/main/sync-repositories.sh